limitador-server/Cargo.toml

```diff
@@ -37,6 +37,8 @@ const_format = "0.2.31"
 lazy_static = "1.4.0"
 clap = "4.3"
 sysinfo = "0.29.7"
+openssl = { version = "0.10.57", features = ["vendored"] }
```

Comment:
- Required updating openssl - https://github.com/Kuadrant/limitador/actions/runs/6573992478/job/17858112986
    - Otherwise will hit this error - https://stackoverflow.com/questions/68871193/pkg-config-error-during-rust-cross-compilation